src/utils/index.ts

```diff
@@ -0,0 +1,7 @@
+export const calculate = {
+  ADD: (prev: number) => (next: number) => prev + next,
+  SUBTRACT: (prev: number) => (next: number) => prev - next,
+  MULTIPLY: (prev: number) => (next: number) => prev * next,
+  DIVIDE: (prev: number) => (next: number) => prev / next,
+  EQUAL: (prev: number) => undefined,
```

Comment:
```suggestion
  EQUAL: (_: number) => undefined,
```
전달은 하고 사용하지 않는다면 언더스코어로 대체해주면 좋아요.

그보다 EQUAL이 undefined를 반환하는 형태로 되어있어서 처음 코드를 보는분들은 왜 반환값을 undefined로 강제해야하는지 맥락파악에 오랜 시간이 들 것 같아요.
다른 코멘트에도 남겼지만 이해되지 않는 맥락이 생길 것 같다면 배경을 설명해주는게 좋아요.

그리고 사소한 부분이지만 undefined보다 null이 더 알맞아보여요.
undefined는 처음 값이 할당되지 않았을때, 그리고 개발자가 의도적으로 빈 값을 표현하고자 할때는 null으로 설정해주는게 의미론적으로 맞는 것 같아요.

Reply:
동의합니다! 여기서 적용해주었어요 [2a5be9f](https://github.com/next-step/react-calculator/pull/4/commits/2a5be9fc3eea7f0b5a0faad19d91206d3cd0ad41) 

사실 EQUAL 에 대해서는 이 객체에 EQUAL이 함께 포함되어 있는게 맞을까? 라는 고민도 했습니다!